### PR TITLE
Pause/Resume gif on image visibility changed

### PR DESCRIPTION
**Pull request by khersonIT**
Chnages related to issue #37.

I encountered the same problem in my project when used your lib. Since my app was used on the slower computers and several gifs was in UI in same time (but only one was visible at once), this fix was critical for me.

It's my first pr from the fork, so let me know if I have to change something.

### CODE REVIEW

File: WpfAnimatedGif/ImageBehavior.cs [hunk 1]
```diff
@@ -237,8 +237,8 @@ public static void RemoveAnimationLoadedHandler(Image image, RoutedEventHandler
             EventManager.RegisterRoutedEvent(
                 "AnimationCompleted",
                 RoutingStrategy.Bubble,
-                typeof (RoutedEventHandler),
-                typeof (ImageBehavior));
+                typeof(RoutedEventHandler),
+                typeof(ImageBehavior));
```
Comment (thomaslevesque) [inline WpfAnimatedGif/ImageBehavior.cs]: Minor issue: please try not to change formatting of unrelated code.
This is a general rule for open-source contributions, not just for this project. It adds noise to the diff, and the project maintainer might not have the same formatting rules as you.
I find the `git add -a` command (interactive add) useful for staging only specific lines in a file. Many GUI tools support this too (SourceTree, GitExtensions, GitKraken...)

File: WpfAnimatedGif/ImageBehavior.cs [hunk 2]
```diff
@@ -300,11 +302,31 @@ private static void AnimatedSourceChanged(DependencyObject o, DependencyProperty
...
+                var controller = GetAnimationController(img);
+                if (controller != null)
+                {
+                    if ((bool)e.NewValue)
+                    {
+                        controller.GotoFrame(0);
+                        controller.Play();
+                    }
+                    else
+                        controller.Pause();
```
Comment (thomaslevesque) [inline WpfAnimatedGif/ImageBehavior.cs]: Minor issue: while I'm not against omitting braces in general, I prefer to keep the `if` and `else` blocks consistent. Could you please add braces there?

**Comment by OmiCron07** [comment]
I use a loading gif on each row of a DataGrid and I experience permanent high CPU usage, so I need this fix too. Hope will be merge and release soon.

Thank you.

**Comment by thomaslevesque** [comment]
@khersonIT I made the requested changes myself so that I could merge. Thanks for the PR!

@OmiCron07 I'm merging it now, and will release ASAP

**Comment by thomaslevesque** [comment]
@OmiCron07 I just released 1.4.18 with this change. It should appear on NuGet soon.

**Pull request merged**